datafusion/functions-aggregate/src/min_max.rs

```diff
@@ -17,21 +17,6 @@
 //! [`Max`] and [`MaxAccumulator`] accumulator for the `max` function
 //! [`Min`] and [`MinAccumulator`] accumulator for the `min` function
 
-// distributed with this work for additional information
```

Comment:
The license header is already in this file -- I am not sure why there is a second partial copy of the header here so I removed it